File: Python/e052.py
'Problem 52 - Permuted multiples'
import time

tic = time.time()
x = 1
while not (sorted(str(2 * x)) == sorted(str(3 * x))
           == sorted(str(4 * x)) == sorted(str(5 * x))
           == sorted(str(6 * x))):
    x += 1
print(x)
toc = time.time()
print(toc - tic)

File: JavaScript/e052.js
// Problem 52: Permuted multiples

/**Sort the number string */
const f = (n) => Array.from(n.toString()).sort().join('');
let x = 1;
while (true) {
    if (f(2 * x) === f(3 * x)
        && f(2 * x) === f(4 * x)
        && f(2 * x) === f(5 * x)
        && f(2 * x) === f(6 * x)) {
        console.log(x); break;
    }
    x++;
}